### DISCUSSION TITLE
how do i enabel resize mode in img2img

### DISCUSSION THREAD

**Discussion by revolverocelot1** [Mikubill/sd-webui-controlnet#2196] [Q&A] [answered]
this feature seems to be removed, i used this for expansion of pixels using inpaint model. how do i reenable it, do i need to go back to old version?

**Comment by huchenlei** [comment, marked as answer]
Use A1111's resize mode selection if you are not passing in an independent control image.

**Reply by revolverocelot1** [reply]
That doesn't seems to work when I use batch process.
